src/yb/tserver/tablet_server_interface.h

```diff
@@ -36,6 +36,7 @@ namespace tserver {
 
 using CertificateReloader = std::function<Status(void)>;
 using PgConfigReloader = std::function<Status(void)>;
+using CQLServerMessenger = std::function<rpc::Messenger*(void)>;
```

Comment:
This seems a convoluted way to get to messenger.

We could just hold/save the cql's messenger, instead of a ptr/callback.